testnet/hardhatdeployer.Dockerfile

```diff
@@ -34,4 +34,5 @@ RUN rm package-lock.json || true
 RUN npm install
 RUN npx hardhat compile
 COPY ./contracts /home/go-obscuro/contracts
+VOLUME /home/go-obscuro/contracts/deployments
```

Comment:
I thought we weren't taking this approach.

If we are, can we add a comment explaining that the Volume is used for mount + test and that params should be passed by either flags or Env vars ? 
Would be preferable to avoid 3 ways to pass starting args and this is the only container that's using it. It's likely in the future we'll make that change to all containers.